src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1077,12 +1077,24 @@ private Task<BatchAccount> CreateBatchAccountAsync()
 
         private Task<IResourceGroup> CreateResourceGroupAsync()
```

Comment:
To avoid duplication:

```
private Task<IResourceGroup> CreateResourceGroupAsync()
{
    var tags = !string.IsNullOrWhiteSpace(configuration.Tags) ? DelimitedTextToDictionary(configuration.Tags, "=", ",") : null;
 
    var resourceGroupDefinition = azureSubscriptionClient
        .ResourceGroups
        .Define(configuration.ResourceGroupName)
        .WithRegion(configuration.RegionName);
        
    resourceGroupDefinition = tags != null ? resourceGroupDefinition.WithTags(tags) : resourceGroupDefinition;
 
    return Execute(
        $"Creating Resource Group: {configuration.ResourceGroupName}...",
        () => resourceGroupDefinition.CreateAsync());
}
```